docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,254 @@
+
+## Connectors
+
+Both the full and lightweight Docker Compose files contain a configuration for [Out-of-the-box Connectors](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md). Refer to the [Connector installation guide](/self-managed/connectors-deployment/install-and-start.md) for details on how to provide the related Connector templates for modeling.
+
+### Connector secrets
+
+Secrets can be added into the Connector runtime using the included `connector-secrets.txt` file. Add secrets in the format `NAME=VALUE`
+per line. The secrets will then be available in the Connector runtime in the format `secrets.NAME`.
+
+### Custom connectors
+
+To add custom Connectors, create a new Docker Image bundling them as described in the [Connectors repository](https://github.com/camunda/connectors).
+
+Alternatively, you can mount new Connector JARs as volumes into the `/opt/app` folder by adding this to the Docker Compose file. Keep in mind that the Connector JARs must include all necessary dependencies inside the JAR.
+
+## Kibana
+
+A [Kibana](https://www.elastic.co/kibana/) profile is available in the provided configuration files to support inspection and exploration of the Camunda 8 data in Elasticsearch.
+
+Enable the profile by adding `--profile kibana` to your Docker Compose command:
+
+```shell
+docker compose --profile kibana up -d
+```
+
+This profile will start Kibana in addition to the default components. Kibana can be used to explore the records exported by Zeebe into Elasticsearch, or to discover the data in Elasticsearch used by the other components (for example, Operate).
+
+Navigate to the Kibana web application and explore the data without login credentials:
+
+- Kibana: [http://localhost:5601](http://localhost:5601)
 
 :::note
-We recommend to use [Helm + KIND](/self-managed/setup/deploy/local/local-kubernetes-cluster.md) instead of Docker Compose for local environments, as the Helm configurations are battle-tested and much closer to production systems.
+
+You need to configure the index patterns in Kibana before you can explore the data.
+
+- Go to `Management > Stack Management > Kibana > Index Patterns`.
+- Create a new index pattern. For example, `zeebe-record-*` matches the exported records.
+  - If you don't see any indexes then make sure to export some data first (e.g. deploy a process). The indexes of the records are created when the first record of this type is exported.
+- Go to `Analytics > Discover` and select the index pattern.
+
 :::
 
-## Web Modeler
+## Troubleshooting
+
+### Running on arm64 based hardware
+
+When using arm64-based hardware (for example, an M1 or M2 Mac), the Keycloak container might not start, as Bitnami only
+provides amd64-based images for versions < 22. You can build and tag an arm-based image locally using the following command:
+
+```shell
+$ DOCKER_BUILDKIT=0 docker build -t bitnami/keycloak:19.0.3 "https://github.com/camunda/camunda-platform.git#8.2.15:.keycloak/"
```

Comment:
Is this still the correct command?

Reply:
It's actually not needed anymore as 8.2 is not supported anymore and the Keycloak versions we use support ARM arch.

I've removed it.